### PR TITLE
Fixes RAM output / Issue #935

### PR DESCRIPTION
**Pull request by b-scholz**

### CODE REVIEW

**Comment by b-scholz** [comment]
This is related to issue #935

File: src/RamStatement.h [hunk 1]
```diff
@@ -776,12 +775,9 @@ class RamLogTimer : public RamStatement {
 
     /** Pretty print */
     void print(std::ostream& os, int tabpos) const override {
-        os << std::string(tabpos, '\t');
-        os << "START_TIMER \"" << stringify(message) << "\"\n";
+        os << times(" ", tabpos) << "START_TIMER \"" << stringify(message) << "\"\n" << std::endl;
```
Comment (mmcgr) [inline src/RamStatement.h]: Is that \n intentional? You removed the others.

**Comment by b-scholz** [comment]
Thanks will fix this.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

File: src/RamStatement.h [hunk 2]
```diff
@@ -982,152 +974,4 @@ class RamLogSize : public RamRelationStatement {
     std::string message;
 };
 
-#ifdef USE_MPI
```
Comment (mmcgr) [inline src/RamStatement.h]: If you're removing MPI, then it would be better in its own commit, own PR, and should remove all the other parts of MPI integration as well.
